onnxscript/eager_mode_evaluator.py

```diff
@@ -92,9 +92,10 @@ def ort_to_os_value(v):
     raise TypeError(f"Unexpected ORT value type {type(v)}.")
 
 
-def call_ort(schema, *args, **kwargs):
+def call_ort(schema, args, kwargs, implicit_args={}):
```

Comment:
implicit_args=None (empty dictionary is not recommended for default value)

Reply:
Done